source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -325,6 +449,8 @@ def make_mock_chaos_item(disruption_name, impacted_obj, impacted_obj_type, start
     impact.severity.id = "SeverityTest"
     impact.severity.wording = "SeverityTest"
     impact.severity.color = "#FFFF00"
+    enums_impact = gtfs_realtime_pb2.Alert.DESCRIPTOR.enum_values_by_name
+    impact.severity.effect = enums_impact["NO_SERVICE"].number if blocking else enums_impact["REDUCED_SERVICE"].number
```

Comment:
actually, chaos only use ```NO_SERVICE``` and ```UNKNOWN_EFFECT```, so it will probably be best to use the same in the tests